### ISSUE TITLE
Basque language support 🎄

### ISSUE THREAD

**Issue by stefan-it** [zalandoresearch/flair#329]
Hi,

I trained a language model for Basque. The available corpora for Basque are quite limited, but I used Wikipedia, OpenSubtitles and two Basque corpora that are usually used for machine translation (more information [here](https://github.com/stefan-it/flair-lms#basque)).

Then I evaluated the `flair` language model on Universal Dependencies (version 1.2) for Basque for PoS tagging.

Additionally, I also trained an ELMo model from scratch (same train split, ~ 2 epochs).

Here are some results (including BERT multilingual embeddings):

| Model/paper | Final Accuracy
| ---------------- | -------------------
| [Yu et. al (2017)](https://arxiv.org/abs/1706.01723)             | 94.94
| [Plank et. al (2016)](https://arxiv.org/abs/1604.05529)        | 95.51
| [Yasunaga et. al (2017)](https://arxiv.org/abs/1711.04903) | 94.71
| FastText Wikipedia embeddings + `flair` language model   | **97.17**
| FastText Wikipedia embeddings + ELMo                             | 95.50
| FastText Wikipedia embeddings + BERT multilingual          | 95.15

Training details are coming in the next days here.

The `flair` forward and backward language models are available from:

```bash
wget https://schweter.eu/cloud/flair-lms/lm-eu-large-forward-v0.1.pt
wget https://schweter.eu/cloud/flair-lms/lm-eu-large-backward-v0.1.pt
```

Feel free to integrate it in the next `flair` version :)

**Comment by alanakbik** [comment]
Hi @stefan-it wow these are great results!! We'll definitely include them in the next release (0.4.1). 

How many words did the combined corpora have? We are getting some questions with regards to minimum corpus size so your experience here would be interesting. 

Also, what dropout are you using when training the LMs? We currently suspect that the default value of 0.5 is way too high since overfitting is not an issue with large corpora, so 0.1 or less might be more appropriate.

Another question: Could you also paste these results in #308 so we have BERT/ELMo/Flair in one place? 

Thanks for sharing the results - really interesting!

**Comment by stefan-it** [comment]
The corpus size was 37,926,708.
 
Thanks for that dropout hint. I retrained the forward model and the ppl was ~ 0.12 lower (validation and test set).

I will add the results in the #308 issue now :)

**Comment by alanakbik** [comment]
close dby #355
